assets/javascripts/kitten/components/typography/sup-title.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import classNames from 'classnames'
+
+export class SupTitle extends React.Component {
+  render() {
+    const { className, children, ...supTitleProps } = this.props
+
+    const supTitleClassName = classNames(
+      'k-SupTitle',
+      className,
+    )
+
+    return (
+      <div className={ supTitleClassName } { ...supTitleProps }>
```

Comment:
Il n'a pas vraiment le rôle d'un paragraphe sémantiquement parlant.
La `div` le rend plus abstrait. Après why not pour lui proposer un tag, ça peut servir pour le SEO.